nntrainer/compiler/ini_interpreter.cpp

```diff
@@ -274,7 +274,6 @@ void IniGraphInterpreter::serialize(const GraphRepresentation &representation,
 
     IniSection s = IniSection::FromExportable(ln->getName(), *ln);
     s.setEntry("type", ln->getType());
-
     sections.push_back(s);
```

Comment:
Please do not add  a style fix in a bugfix commit. Remove this file change from the commit.
You will confuse yourself of the future with this commit if you track git log based on filenames in the future.